zip-0207.rst

```diff
@@ -0,0 +1,277 @@
+::
+
+  ZIP: 207
+  Title: Split Founders' Reward
+  Author: Jack Grigg <jack@z.cash>
+  Category: Consensus
+  Created: 2019-01-04
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+
+Abstract
+========
+
+This proposal defines how the consensus rules are altered to split the original Founders' Reward across three
+recipient addresses per block instead of one, corresponding to the three funding streams contained within it.
+
+
+Motivation
+==========
+
+Since the launch of the Zcash network, the consensus rules have required that until the first block reward
+halving (at block 850,000), each block must send 20% of the block subsidy to a hard-coded transparent address.
+[#block-subsidy]_ This stream of 2.5-ZEC outputs (the value after the mining slow-start was completed) can be
+split into three logical funding streams: [#continued-funding]_
+
+- (73.7%) The Founders' Reward stream
+- (14.4%) The Zcash Foundation Endowment stream
+- (11.9%) The Zerocoin Electric Coin Company Strategic Reserve stream
+
+Modifying the consensus rules to allocate the 2.5 ZEC across three separate recipient addresses decouples
+these three funding streams organizationally, legally, and operationally. It further reinforces transparency
+as to the structure of the original Founders' Reward.
+
+
+Specification
+=============
+
+Funding streams
+---------------
+
+A funding stream is defined by a constant value per block (in zatoshis), a start height (inclusive), and an
+end height (exclusive). The start height MUST be after the end of the mining slow-start (to make specification
+of the value simpler).
+
+- To-do: Decide whether to use fractional value per block instead, which gets around the mining slow-start
+  problem (and halvings) but might make implementation more complex.
```

Comment:
Given that we intend to alter the block target time & issuance-per-block (to have faster blocks while keeping a consistent issuance schedule based on time), it is "cleaner" to define the spec by fractional values of the total block issuance. Also, in terms of code, it may help to use the same abstraction.

However, if that's too complex, then we need to specify both this and the changes block rate so that they dovetail together. Also, any future change to block rate during the active period of this spec requires a further modification, though I don't anticipate any.